shared/types/submission.ts

```diff
@@ -101,17 +124,40 @@ export const SubmissionPaymentDto = z.object({
 })
 export type SubmissionPaymentDto = z.infer<typeof SubmissionPaymentDto>
 
-export type StorageModeSubmissionDto = {
+export type SubmissionDtoBase = {
+  submissionType: SubmissionType.Encrypt | SubmissionType.Multirespondent
   refNo: SubmissionId
   submissionTime: string
+}
+
+export type StorageModeSubmissionDto = SubmissionDtoBase & {
+  submissionType: SubmissionType.Encrypt
   content: string
   verified?: string
   attachmentMetadata: Record<string, string>
   payment?: SubmissionPaymentDto
   version: number
 }
 
+export type MultirespondentSubmissionDto = SubmissionDtoBase & {
```

Comment:
Similar to above - can we extend `StorageModeSubmissionDto` instead if we are aiming for feature parity in the final version?